### PR TITLE
Tentative merge of SWIG master

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

File: taskcluster/tc-dotnet-utils.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ install_nuget()
 
   ls -halR "${PROJECT_NAME}.${DS_VERSION}"
 
-  nuget install NAudio
+  nuget install NAudio -Version 1.10.0
```
Comment (reuben) [inline taskcluster/tc-dotnet-utils.sh:37]: Hm. I wonder if this dependency and version spec should be in some project file to be installed automatically. Seems weird to have it just laying here. Maybe add a TODO comment? Right now is not the time to chase this I guess.

**Review by reuben — APPROVED** [review]

File: taskcluster/tc-dotnet-utils.sh (hunk 1 above)
Comment (lissyx) [reply taskcluster/tc-dotnet-utils.sh:37]: Yeah, I agree and I'm unsure as well, so I'll keep #3540 open after we fix this and wait for carl's feedback on that point.

**Comment by lissyx** [comment]
@dsteinman This will make ElectronJS newer versions easier, hopefully :)

**Comment by dsteinman** [comment]
That's great news!

I haven't been too busy on my voice project in a while but at some point I'll get back to it.

**Pull request merged**